lib/modules/datasource/nuget/v3.ts

```diff
@@ -91,7 +91,7 @@ export async function getResourceUrl(
     }
     logger.debug(
       { err, url },
-      `nuget registry failure: can't get ${resourceType}`
+      `nuget registry failure: can't get ${resourceType} of servicesIndexRaw ${servicesIndexRaw}`
```

Comment:
Log it as metadata

Reply:
@rarkins since I am not used to typescript, can you give me a little hint to an example where metadata is already logged?

Reply:
Next to err and url

Reply:
Should be solved now.